js/components/operatorSignIn/text.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+
+// 🚨 You MUST change the version if you change the text!
+// 🚨 You MUST update the document at https://www.notion.so/mbta-downtown-crossing/Attestation-Text-History-7f7117d7c4574013a48c6819c7809c4a?pvs=4
+export const useSignInText = (): {
```

Comment:
**suggestion (non-blocking):** I don't think this actually needs to be a hook currently, especially since we haven't implemented any time-based functionality for Orbit yet. But I also don't think it hurts anything.